shoop/core/models/products.py

```diff
@@ -243,7 +242,7 @@ def get_shop_instance(self, shop):
         if cached:
             return cached
 
-        shop_inst = self.shop_products.filter(shop=shop).first()
+        shop_inst = self.shop_products.get(shop=shop)
         if shop_inst:
```

Comment:
also wont this fail miserably if there are multiple shops and products for them?